### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by shrutivd**
**Code changes I made:**
1. Added answers to css dinner game in 01-selectors.md file.
2. Added code in js to check if the number is palindrome and associated it with the HTML code
3. Added bootstrap classes for formatting the page as required

**Why I made the changes:**
The assignment asked to complete all the levels of css dinner game and enlist the selectors in the file. The second task was to check if the number entered by the user is positive and if so, check whether it is a palindrome. The user must be provided with proper messages for the number entered. 
This is the reason that I completed these tasks in this assignment. 
 
**What feedback I am looking for:**
1. Suggest if the selectors I used are inefficient or if any good alternatives are available for them.
2. Suggest the bootstrap classes if you find mistakes in what I have used.
3. Suggest the styling corrections according to the Airbnb styling guide.
4. Suggest changes in js file that you find are necessary.

### CODE REVIEW

File: hw1/02-palindrome.html
Comment (NathanMoes) [inline hw1/02-palindrome.html:1]: I would sugest reducing the tab size in number of spaces for the auto format for html. This seems a bit much space.

File: hw1/.eslintrc.js
Comment (NathanMoes) [inline hw1/.eslintrc.js:1]: Smart! I will try to add this to my project as well if it help

**Comment by NathanMoes** [comment]
2 Things I would change here. 
1: I believe the style guide says to give strings single quotes intstead of doubble aka 'quote' instead of "quote"
2: a I would use the css classes instead of the css color for the valid/invalid input. b: When you do a valid palindrom and enter a invalid number (e - eulers number I guess) it will have green text saying there is an invaild number. Aka green text with error. Thus I also sugested the classes, as you can add and remove classes such as I did on my hw1. 
Other than that, looks good! I used an in-place algo for the palindrome but your copy compare method works as well

**Review by NathanMoes — APPROVED** [review]

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+		.map((num) => {
+			return Number(num);
+		});
+	//clone the array and reverse it
+	const new_arr = [...myArr].reverse();
+	if (myArr.toString() === new_arr.toString()) {
+		return true;
+	} else {
+		return false;
+	}
```
Comment (halltim91) [inline hw1/02-palindrome.js]: This could be simplified to just return the result of the comparison instead

**Review by halltim91 — APPROVED** [review]

File: hw1/02-palindrome.html
Comment (shrutivd) [reply hw1/02-palindrome.html:1]: I am using prettier to format both HTML and JavaScript. I will have to look for space settings starting the next assignment as the issue is not related to functionality or implementation standards.

**Comment by shrutivd** [comment]
> 2 Things I would change here. 1: I believe the style guide says to give strings single quotes intstead of doubble aka 'quote' instead of "quote" 2: a I would use the css classes instead of the css color for the valid/invalid input. b: When you do a valid palindrom and enter a invalid number (e - eulers number I guess) it will have green text saying there is an invaild number. Aka green text with error. Thus I also sugested the classes, as you can add and remove classes such as I did on my hw1. Other than that, looks good! I used an in-place algo for the palindrome but your copy compare method works as well

Thanks for the review Nathan, 
I have made changes as you suggested. However, I did not add classes for changing text color. I will practice that from the next assignment.

File: hw1/02-palindrome.js (hunk 1 above)
Comment (shrutivd) [reply hw1/02-palindrome.js]: Thanks Tim, 
 I have made the changes accordingly

**Pull request merged**